package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nimblehq/infra-template",
-  "version": "1.2.0",
+  "version": "1.4.0",
```

Comment:
is it 1.3.0 @hoangmirs

Reply:
`1.3.0` was the last release. I did not change it, then I had to release it manually